### ISSUE TITLE
ONNX Post-Training Quantization（semantic segmentation task）

### ISSUE THREAD

**Issue by Masque555** [openvinotoolkit/openvino_notebooks#1223]
nncf is a very useful tool. But I didn't find any example about segmentation of quantized onnx after training, can you provide some examples of semantic segmentation? Such as VOC dataset, Cityspace dataset.

**Comment by siddharth062022** [comment]
i have also same issue how to use nncf direct no ui based system.

**Comment by alexsu52** [comment]
Hello,

You can find notebook to quantize segmentation model here: https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/110-ct-segmentation-quantize

I highly recommend using OpenVINO model to obtain quantized model with the best performance in OpenVINO.

**Comment by Masque555** [comment]
> Hello,
> 
> You can find notebook to quantize segmentation model here: https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/110-ct-segmentation-quantize
> 
> I highly recommend using OpenVINO model to obtain quantized model with the best performance in OpenVINO.

Thank for the reply, I will try it.

**Comment by ch1x0r** [comment]
It works on a local Linux. I used mint and an AMD processor. If you take the segmentation documentation steps from the project, this works local no issues. I would  check out yolo8 seg demo docs

**Comment by avitial** [comment]
Closing this, I hope previous responses were sufficient to help you proceed. Feel free to reopen and ask additional questions related to this topic.

**Comment by Masque555** [comment]
> i have also same issue how to use nncf direct no ui based system.

import nncf
import openvino.runtime as ov
import torch
from torchvision import datasets, transforms
import openvino.runtime as ov


model = ov.Core().read_model("./model4/end2end.xml")

train_transforms = transforms.Compose([
    transforms.Resize((512, 512)),
    transforms.RandomCrop((512, 512)),
    transforms.RandomHorizontalFlip(),
    transforms.ColorJitter(brightness=0.2, contrast=0.2, saturation=0.2, hue=0.1),
    transforms.ToTensor(),
    transforms.Normalize(mean=[0.47, 0.45, 0.44], std=[0.37, 0.35, 0.34]),
])


#calibration_dataset = ImageFolder("./calibration_dataset", transform=train_transforms)


val_dataset = datasets.ImageFolder("./FNDataSet", transform=train_transforms )
dataset_loader = torch.utils.data.DataLoader(val_dataset, batch_size=1)

def transform_fn(data_item):
    images, _ = data_item
    return images

calibration_dataset = nncf.Dataset(dataset_loader, transform_fn)

quantized_model = nncf.quantize(model, calibration_dataset)

ov.serialize(quantized_model, "./model4/INT8_FN.xml")

I hope it can help you
